core/src/main/java/org/frankframework/util/MessageUtils.java

```diff
@@ -373,4 +431,60 @@ public static String asString(Object object) throws IOException {
 			return message.asString();
 		}
 	}
+
+	/**
+	 * Convert input value to a message in JSON format and mimetype. If the input value is already JSON, then it
+	 * is returned as-is. If the value is in XML format, it will be converted to JSON using {@link UtilityTransformerPools#getXml2JsonTransformerPool()}.
+	 * Otherwise the string-value of the input-value will be wrapped as JSON as {@code {"value": value}}.
+	 */
+	public static @Nonnull Message convertToJsonMessage(@Nonnull Object value) throws IOException, XmlException {
+		return convertToJsonMessage(value, "value");
+	}
+
+	/**
+	 * Convert input value to a message in JSON format and mimetype. If the input value is already JSON, then it
+	 * is returned as-is. If the value is in XML format, it will be converted to JSON using {@link UtilityTransformerPools#getXml2JsonTransformerPool()}.
+	 * Otherwise the string-value of the input-value will be wrapped as JSON as {@code {"valueName": value}}, using parameter {@code valueName} as
+	 * name of the object.
+	 */
+	public static @Nonnull Message convertToJsonMessage(@Nonnull Object value, @Nonnull String valueName) throws IOException, XmlException {
+		Message message = Message.asMessage(value);
+		message.preserve();
+		MimeType mimeType = MessageUtils.computeMimeType(message);
+		if (MediaType.APPLICATION_JSON.isCompatibleWith(mimeType)) {
+			return message;
+		}
+
+		if (MediaType.APPLICATION_XML.isCompatibleWith(mimeType)) {
+			try {
+				TransformerPool tpXml2Json = UtilityTransformerPools.getXml2JsonTransformerPool();
+				Map<String, Object> parameterValues = Collections.singletonMap("includeRootElement", true);
+				Message result = tpXml2Json.transform(message, parameterValues);
+				result.getContext().withMimeType(MediaType.APPLICATION_JSON);
```

Comment:
Eigenlijk moet in deze methode (transform) die withMimeType zitten :)

Reply:
Maar dat is een algemene transform die text of XML geeft, afhankelijk van wat voor Stylesheet het is... Kan je in de TransformerPool al meegeven wat het output mimetype moet zijn?
Moet ik even naar kijken, of ik daar uitkom.